#!/home/danilodelucio/Desktop/test_python/new_python/bin/python3.11
import sys
from lib2to3.main import main

sys.exit(main("lib2to3.fixes"))
